### ISSUE TITLE
An unknown server-side error occurred while processing the command. Original error: '12.1' does not exist in the list of simctl SDKs. No Simulator SDK versions are available on your system. Please install some via Xcode preferences.

### ISSUE THREAD

**Issue by parikhparth23** [appium/appium#12391]
## The problem

Getting this error:
"An unknown server-side error occurred while processing the command. Original error: '12.1' does not exist in the list of simctl SDKs. No Simulator SDK versions are available on your system. Please install some via Xcode preferences."

when running appium Version 1.10.0 (1.10.0.20181230.2)

When i run xcrun simctl list i can see 12.1 part of it.

== Device Types ==
iPhone 4s (com.apple.CoreSimulator.SimDeviceType.iPhone-4s)
iPhone 5 (com.apple.CoreSimulator.SimDeviceType.iPhone-5)
iPhone 5s (com.apple.CoreSimulator.SimDeviceType.iPhone-5s)
iPhone 6 (com.apple.CoreSimulator.SimDeviceType.iPhone-6)
iPhone 6 Plus (com.apple.CoreSimulator.SimDeviceType.iPhone-6-Plus)
iPhone 6s (com.apple.CoreSimulator.SimDeviceType.iPhone-6s)
iPhone 6s Plus (com.apple.CoreSimulator.SimDeviceType.iPhone-6s-Plus)
iPhone 7 (com.apple.CoreSimulator.SimDeviceType.iPhone-7)
iPhone 7 Plus (com.apple.CoreSimulator.SimDeviceType.iPhone-7-Plus)
iPhone 8 (com.apple.CoreSimulator.SimDeviceType.iPhone-8)
iPhone 8 Plus (com.apple.CoreSimulator.SimDeviceType.iPhone-8-Plus)
iPhone SE (com.apple.CoreSimulator.SimDeviceType.iPhone-SE)
iPhone X (com.apple.CoreSimulator.SimDeviceType.iPhone-X)
iPhone Xs (com.apple.CoreSimulator.SimDeviceType.iPhone-XS)
iPhone Xs Max (com.apple.CoreSimulator.SimDeviceType.iPhone-XS-Max)
iPhone Xʀ (com.apple.CoreSimulator.SimDeviceType.iPhone-XR)
iPad 2 (com.apple.CoreSimulator.SimDeviceType.iPad-2)
iPad Retina (com.apple.CoreSimulator.SimDeviceType.iPad-Retina)
iPad Air (com.apple.CoreSimulator.SimDeviceType.iPad-Air)
iPad Air 2 (com.apple.CoreSimulator.SimDeviceType.iPad-Air-2)
iPad (5th generation) (com.apple.CoreSimulator.SimDeviceType.iPad--5th-generation-)
iPad Pro (9.7-inch) (com.apple.CoreSimulator.SimDeviceType.iPad-Pro--9-7-inch-)
iPad Pro (12.9-inch) (com.apple.CoreSimulator.SimDeviceType.iPad-Pro)
iPad Pro (12.9-inch) (2nd generation) (com.apple.CoreSimulator.SimDeviceType.iPad-Pro--12-9-inch---2nd-generation-)
iPad Pro (10.5-inch) (com.apple.CoreSimulator.SimDeviceType.iPad-Pro--10-5-inch-)
iPad (6th generation) (com.apple.CoreSimulator.SimDeviceType.iPad--6th-generation-)
iPad Pro (11-inch) (com.apple.CoreSimulator.SimDeviceType.iPad-Pro--11-inch-)
iPad Pro (12.9-inch) (3rd generation) (com.apple.CoreSimulator.SimDeviceType.iPad-Pro--12-9-inch---3rd-generation-)
Apple TV (com.apple.CoreSimulator.SimDeviceType.Apple-TV-1080p)
Apple TV 4K (com.apple.CoreSimulator.SimDeviceType.Apple-TV-4K-4K)
Apple TV 4K (at 1080p) (com.apple.CoreSimulator.SimDeviceType.Apple-TV-4K-1080p)
Apple Watch - 38mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-38mm)
Apple Watch - 42mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-42mm)
Apple Watch Series 2 - 38mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-Series-2-38mm)
Apple Watch Series 2 - 42mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-Series-2-42mm)
Apple Watch Series 3 - 38mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-Series-3-38mm)
Apple Watch Series 3 - 42mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-Series-3-42mm)
Apple Watch Series 4 - 40mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-Series-4-40mm)
Apple Watch Series 4 - 44mm (com.apple.CoreSimulator.SimDeviceType.Apple-Watch-Series-4-44mm)
== Runtimes ==
iOS 12.1 (12.1 - 16B91) - com.apple.CoreSimulator.SimRuntime.iOS-12-1 
tvOS 12.1 (12.1 - 16J602) - com.apple.CoreSimulator.SimRuntime.tvOS-12-1 
watchOS 5.1 (5.1 - 16R591) - com.apple.CoreSimulator.SimRuntime.watchOS-5-1 
== Devices ==
-- iOS 12.1 --
    iPhone 5s (CF4D8F85-F773-473C-8374-FFBD286DC46C) (Shutdown) 
    iPhone 6 (DE292AF8-0C6E-47EF-83B0-5865376CFEC2) (Shutdown) 
    iPhone 6 Plus (55C2DA6D-E2A6-46C4-B8C6-16763E58051C) (Shutdown) 
    iPhone 6s (7382AE6F-E26D-46F6-8857-C0A427CD03C9) (Shutdown) 
    iPhone 6s Plus (A3E28226-CEDC-4B6C-970A-13362DE91928) (Shutdown) 
    iPhone 7 (F020CF54-09C4-4E46-B4F3-7A42794EB118) (Shutdown) 
    iPhone 7 Plus (B9E2BC74-59B7-4177-A28D-C4C67125DDD3) (Shutdown) 
    iPhone 8 (8C78231F-C9E3-4771-B26A-CF35BE929B0A) (Shutdown) 
    iPhone 8 Plus (32972038-190A-4392-B394-F5F6D1FFC240) (Shutdown) 
    iPhone SE (C19CD3B6-EB89-45CF-A6AC-777EDEAD2642) (Shutdown) 
    iPhone X (26D0C477-36FB-421C-8CA8-9891F5380D81) (Shutdown) 
    iPhone XS (A07B8ED5-939A-4ED8-912B-18072AFCB09A) (Shutdown) 
    iPhone XS Max (322ADC01-0A00-41E2-8956-5000FC3C35C7) (Shutdown) 
    iPhone XR (2FAF7F78-32D4-4D8B-A044-2213B9B2DA28) (Shutdown) 
    iPad Air (7B6CAE6F-A9AE-4AF9-BBF6-5E697D3CD074) (Shutdown) 
    iPad Air 2 (D9385BE8-6F54-4F9C-8F86-88613AEC7EFE) (Shutdown) 
    iPad (5th generation) (1C3D8E14-CD24-4F8D-894B-4B0B7AADEEC0) (Shutdown) 
    iPad Pro (9.7-inch) (5329E861-E34B-41FE-8138-E5676388C11A) (Shutdown) 
    iPad Pro (12.9-inch) (0402A969-2D9F-4265-BA9F-DA5E449B066C) (Shutdown) 
    iPad Pro (12.9-inch) (2nd generation) (31B4653C-56EC-4277-A722-9948679EE0A8) (Shutdown) 
    iPad Pro (10.5-inch) (BCE50226-39E7-4632-B2C0-34E82B9F9F49) (Shutdown) 
    iPad (6th generation) (43844525-5A68-4513-B5F4-EA0CB545C7F0) (Shutdown) 
    iPad Pro (11-inch) (51DAD0A4-64D9-44BD-9FE6-7F7617C7F9AC) (Shutdown) 
    iPad Pro (12.9-inch) (3rd generation) (592DD2A9-4E39-4906-B640-C6A8EE278B35) (Shutdown) 
-- tvOS 12.1 --
    Apple TV (7142C0B1-3951-4AC7-81DC-EFB3D1502668) (Shutdown) 
    Apple TV 4K (5EE04723-5D98-436B-A198-E5F0E7DC7C1D) (Shutdown) 
    Apple TV 4K (at 1080p) (325EBB89-98DF-4ECF-9D66-17F25F906F43) (Shutdown) 
-- watchOS 5.1 --
    Apple Watch Series 2 - 38mm (8DA8B4E2-8790-4502-BDAC-2A9740F17553) (Shutdown) 
    Apple Watch Series 2 - 42mm (64D87B49-BC7A-41E1-84C0-AB3D1F233173) (Shutdown) 
    Apple Watch Series 3 - 38mm (01CCEFBB-2EA3-40F4-85A9-325B0A6EF3FD) (Shutdown) 
    Apple Watch Series 3 - 42mm (0EC6C995-D0BA-4D35-B971-FEEA109E238F) (Shutdown) 
    Apple Watch Series 4 - 40mm (31111D80-8735-4850-8EB0-15F08AD3495C) (Shutdown) 
    Apple Watch Series 4 - 44mm (6EDEF653-9714-4BD5-ACB8-2045F7C0FD80) (Shutdown) 
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-11-3 --
    iPhone 7 (49455E8F-7DFF-4A4B-A651-DDD6725CEC37) (Creating) (unavailable, runtime profile not found)
    iPhone 7 Plus (7C1B2138-3E63-4209-96FA-456EF5D652FA) (Creating) (unavailable, runtime profile not found)
    iPhone 8 (E72C1CE2-7163-4D12-B15A-99C42C824422) (Creating) (unavailable, runtime profile not found)
    iPhone 8 Plus (4D9BA719-B28E-4215-B687-410B137E0021) (Creating) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-12-0 --
    iPhone 5s (03A63F37-0CEE-434D-A8B0-6DC5D691F4A5) (Shutdown) (unavailable, runtime profile not found)
    appiumTest-iPhone 6 (C8478825-386D-465A-BF90-396192FED7A8) (Shutdown) (unavailable, runtime profile not found)
    iPhone 7 (9EEF757B-4F22-42AC-A270-7B73300CD8D5) (Shutdown) (unavailable, runtime profile not found)
    iPhone 7 Plus (4039BD88-AE66-47CA-88AA-8CFF92FF8C4C) (Shutdown) (unavailable, runtime profile not found)
    iPhone 8 (D7638B37-68E9-45F2-B275-067AD5568776) (Shutdown) (unavailable, runtime profile not found)
    iPhone 8 Plus (2D2ADFCA-3675-495A-816A-7903BA6971DE) (Shutdown) (unavailable, runtime profile not found)
    iPhone SE (3657F11D-A6C9-4360-96CB-D0F4394E6422) (Shutdown) (unavailable, runtime profile not found)
    iPhone X (BB38C217-7775-4A10-A8E3-158887AB2801) (Shutdown) (unavailable, runtime profile not found)
    iPhone XS (C7BE2ED2-7361-4A37-B4D6-DF3B621056BB) (Shutdown) (unavailable, runtime profile not found)
    expensiveiphone (CD8F3107-A909-4808-B2F3-89C66C2D5490) (Shutdown) (unavailable, runtime profile not found)
    iPhoneXSMax (EA7AAE98-3F5F-42A9-B7C6-E0E5DF2592E3) (Shutdown) (unavailable, runtime profile not found)
    iPhone XR (CDD34403-F314-4685-9A96-0B9B5B9349DC) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (F6F11E14-A5E6-4728-9AA0-60A7D4C6F2CF) (Shutdown) (unavailable, runtime profile not found)
    iPad Air 2 (5C67E462-7EB2-402D-A97C-B1F696E1DB72) (Shutdown) (unavailable, runtime profile not found)
    iPad (5th generation) (538B6E36-6A4B-4991-B91A-6CC87A7B1777) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (9.7-inch) (0A154314-A8E8-4048-A88D-CFB7ECCF58E1) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (12.9-inch) (C94717A4-19E8-4147-B71A-13E9B99DE011) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (12.9-inch) (2nd generation) (B59F21F9-E39A-4115-A55B-889069A9F52F) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (10.5-inch) (376443B8-9662-4D74-94E2-0B36DD451752) (Shutdown) (unavailable, runtime profile not found)
    iPad (6th generation) (E9A0E181-156E-4994-8B99-F3C63BAB05B8) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.iOS-12-2 --
    iPhone 5s (3A432FEC-51E8-4A33-9845-10F1CF730F6F) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 (602051BF-8B00-4C5F-8264-9799BBE82054) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6 Plus (9BF5C8DE-87D6-4DA8-97F0-36C04B6742FA) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6s (4070580E-6B5F-4A99-85B8-8611599A6ED9) (Shutdown) (unavailable, runtime profile not found)
    iPhone 6s Plus (B75C633B-FD04-4C22-9726-7FD303BC8F60) (Shutdown) (unavailable, runtime profile not found)
    iPhone 7 (829C36C1-27C9-4645-BB6C-6CB0BACDA456) (Shutdown) (unavailable, runtime profile not found)
    iPhone 7 Plus (F7567346-0CD8-49FD-AF65-DF614D47D13F) (Shutdown) (unavailable, runtime profile not found)
    iPhone 8 (4734FCC0-50D2-4D60-A434-40A9292AEC8B) (Shutdown) (unavailable, runtime profile not found)
    iPhone 8 Plus (8F016CBE-EB12-4299-931B-BDA86F8466BC) (Shutdown) (unavailable, runtime profile not found)
    iPhone SE (7AA31682-A1B6-439E-8C61-657CE32D8405) (Shutdown) (unavailable, runtime profile not found)
    iPhone X (DE268E8B-0F84-4127-A3E8-087A7FA71975) (Shutdown) (unavailable, runtime profile not found)
    iPhone Xs (6AD5779E-BECA-4B81-A064-F5EC23322494) (Shutdown) (unavailable, runtime profile not found)
    iPhone Xs Max (958C4628-8FC4-484E-B8A9-DB85C117B446) (Shutdown) (unavailable, runtime profile not found)
    iPhone Xʀ (4A0F5079-BB8D-4A8F-81F8-02917FFEB34A) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (3BFE78D6-5518-4605-8A1E-0F46B6256486) (Shutdown) (unavailable, runtime profile not found)
    iPad Air 2 (0D8572FD-A617-4B1D-ADA7-7015FDF48675) (Shutdown) (unavailable, runtime profile not found)
    iPad (5th generation) (9948BD8F-AB8A-4E11-9660-398C0AFA21F3) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (9.7-inch) (974A5EB4-668D-4FA7-B7CF-E332A30B61F6) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (12.9-inch) (468E8288-E358-4EE1-82D0-609D941FF35A) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (12.9-inch) (2nd generation) (67D4692C-69CC-4878-B2D1-A2006D7C63BC) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (10.5-inch) (AAD77B77-DF49-4109-8752-FB39B348694F) (Shutdown) (unavailable, runtime profile not found)
    iPad (6th generation) (3B136739-D972-4215-9EB1-988B355D03DA) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (11-inch) (A3712F04-943C-4666-8C25-B2FABB1D6479) (Shutdown) (unavailable, runtime profile not found)
    iPad Pro (12.9-inch) (3rd generation) (0D46C463-A5E6-4116-87D1-1112CED84576) (Shutdown) (unavailable, runtime profile not found)
    iPad Air (3rd generation) (E0B99ECD-5ABA-4083-9206-A10C0095437E) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.tvOS-12-0 --
    Apple TV (AEF45017-D47B-4539-A3BE-D624656931BC) (Shutdown) (unavailable, runtime profile not found)
    Apple TV 4K (966450D5-8F5B-4F5D-A5D6-BD8EDEDC101C) (Shutdown) (unavailable, runtime profile not found)
    Apple TV 4K (at 1080p) (19BB1A49-568B-46D3-979B-9AB1E9A8F05D) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.tvOS-12-2 --
    Apple TV (9405EDFB-4BF4-45E4-B034-42A857EE8FC5) (Shutdown) (unavailable, runtime profile not found)
    Apple TV 4K (5D2FB4BA-A460-4123-874E-A22FB96CA278) (Shutdown) (unavailable, runtime profile not found)
    Apple TV 4K (at 1080p) (8F65E7B8-7EA7-4FA1-96C5-6502790B2026) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.watchOS-4-2 --
    Apple Watch Series 2 - 38mm (5CC7E54C-CED3-4004-BA17-6EC82B0A64D7) (Creating) (unavailable, runtime profile not found)
    Apple Watch Series 2 - 42mm (05205C70-B5C8-4964-8717-BA52E2DE43BA) (Creating) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 38mm (7EA5F6CF-A5AF-4127-9B54-A7800ECE3183) (Creating) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 42mm (BD63E13B-FBFA-4112-95E3-E7A9F0F0375D) (Creating) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.watchOS-4-3 --
    Apple Watch Series 2 - 38mm (F38B7B24-7D56-4AC1-A4C1-88D230CD5544) (Creating) (unavailable, runtime profile not found)
    Apple Watch Series 2 - 42mm (A9943D65-3EB6-40C0-8EB0-FC34F919E9B9) (Creating) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 38mm (F3FF2136-589F-4F08-94C5-DB98501426D1) (Creating) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 42mm (E1D527A4-D5C2-4501-BC40-D9E32EEA79E9) (Creating) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.watchOS-5-0 --
    Apple Watch Series 2 - 38mm (F8F57E3B-B40D-4556-A1FD-1FD573D5754C) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 2 - 42mm (4F0A5BA5-985A-490A-B20E-F60329064554) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 38mm (0E722197-77EA-47EF-8D80-0ABEB05D0BCF) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 42mm (82081C53-F671-4572-A4A0-0CB317E4A583) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 4 - 40mm (167E631B-BFE5-4BB5-9D84-15B14B46B160) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 4 - 44mm (7939D818-BBD8-4CF6-94DD-79C5C72CA66A) (Shutdown) (unavailable, runtime profile not found)
-- Unavailable: com.apple.CoreSimulator.SimRuntime.watchOS-5-2 --
    Apple Watch Series 2 - 38mm (709F7FDC-08AA-4ADF-BC4E-ADB6CFB36DDD) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 2 - 42mm (8FD1DAEA-9D59-44A9-BD99-6078A193B38D) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 38mm (C63DB3FE-16B1-44AD-9F3D-FB2E1BBAB4EB) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 3 - 42mm (B2943BFA-270B-4A78-9AB2-2510450783F7) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 4 - 40mm (7C0C3A7E-9465-49DC-AB3E-1D6D2F5F0D42) (Shutdown) (unavailable, runtime profile not found)
    Apple Watch Series 4 - 44mm (797A40E8-8CDE-4D80-BB38-ACE4070A2150) (Shutdown) (unavailable, runtime profile not found)
== Device Pairs ==
88B967B7-9F32-4D61-88CE-E12B173AEE3F (active, disconnected)
    Watch: Apple Watch Series 4 - 40mm (31111D80-8735-4850-8EB0-15F08AD3495C) (Shutdown)
    Phone: iPhone XS (A07B8ED5-939A-4ED8-912B-18072AFCB09A) (Shutdown)
6765E74A-5056-4628-A4E1-101ED2E0B9AE (active, disconnected)
    Watch: Apple Watch Series 4 - 44mm (6EDEF653-9714-4BD5-ACB8-2045F7C0FD80) (Shutdown)
    Phone: iPhone XS Max (322ADC01-0A00-41E2-8956-5000FC3C35C7) (Shutdown)
0C01311F-7F95-47C1-947F-A022904CE971 (unavailable)
    Watch: Apple Watch Series 4 - 40mm (167E631B-BFE5-4BB5-9D84-15B14B46B160) (Shutdown)
    Phone: iPhone XS (C7BE2ED2-7361-4A37-B4D6-DF3B621056BB) (Shutdown)
B818F851-4004-4A9F-A27C-E3B75E3E334B (unavailable)
    Watch: Apple Watch Series 4 - 44mm (7939D818-BBD8-4CF6-94DD-79C5C72CA66A) (Shutdown)
    Phone: expensiveiphone (CD8F3107-A909-4808-B2F3-89C66C2D5490) (Shutdown)
E4AEF2D6-D018-4BEB-9FA5-A4704C9AA07C (unavailable)
    Watch: Apple Watch Series 4 - 40mm (7C0C3A7E-9465-49DC-AB3E-1D6D2F5F0D42) (Shutdown)
    Phone: iPhone Xs (6AD5779E-BECA-4B81-A064-F5EC23322494) (Shutdown)
2B91F14A-648A-4707-8C28-060DE2F3A934 (unavailable)
    Watch: Apple Watch Series 4 - 44mm (797A40E8-8CDE-4D80-BB38-ACE4070A2150) (Shutdown)
    Phone: iPhone Xs Max (958C4628-8FC4-484E-B8A9-DB85C117B446) (Shutdown)

**Comment by parikhparth23** [comment]
i am using xcode version 10.1

**Comment by parikhparth23** [comment]
#appium #xcode

**Comment by KazuCocoa** [comment]
Can you observe below output by `xcrun simctl list -j`? The format of `"com.apple.CoreSimulator.SimRuntime.iOS-11-4"` part has been changed in Xcode 10.2.

```
  "devices" : {
    "com.apple.CoreSimulator.SimRuntime.iOS-11-4" : [
      {
        "availability" : "(available)",
        "state" : "Shutdown",
        "isAvailable" : true,
        "name" : "iPhone 5s",
        "udid" : "981CDAC8-CC8E-4E5C-8939-8944A7F0E523",
        "availabilityError" : ""
      },
```

If you installed Xcode 10.2 once (include beta), the machine could return new format even you use Xcode 10.1 with xcode-select (It happened on my machine)

**Comment by parikhparth23** [comment]
@KazuCocoa i can see the following:
` "devices" : {
    "com.apple.CoreSimulator.SimRuntime.tvOS-12-1" : [

    ],
    "com.apple.CoreSimulator.SimRuntime.iOS-12-1" : [
      {
        "availability" : "(available)",
        "state" : "Shutdown",
        "isAvailable" : true,
        "name" : "iPhone Xs Max",
        "udid" : "37976EDA-1D6E-440B-B3B2-2D9B44721A1E",
        "availabilityError" : ""
      }
    ],`

**Comment by parikhparth23** [comment]
@KazuCocoa what should i do?

**Comment by KazuCocoa** [comment]
OK.
Then, you need to update to over 1.11.
1.12.1 has some fixes for iOS12. 2

**Comment by parikhparth23** [comment]
does 1.12.1 have a  dmg file?

**Comment by KazuCocoa** [comment]
Do you mean Appium desktop?
Maybe not yet. You can use Appium 1.12.0 ver.

**Comment by parikhparth23** [comment]
i dont see 1.12.0 ver dmg file https://github.com/appium/appium/releases/tag/v1.12.0

**Comment by KazuCocoa** [comment]
1. Install appium via npm install following http://appium.io/docs/en/about-appium/getting-started/
2. launch the appium server
3. Launch desktop app
4. Open inspector view
5. Connect to the running appium serve via Custom Server tab in the inspector view

**Comment by jlee6890** [comment]
> 1. Install appium via npm install following http://appium.io/docs/en/about-appium/getting-started/
> 2. launch the appium server
> 3. Launch desktop app
> 4. Open inspector view
> 5. Connect to the running appium serve via Custom Server tab in the inspector view

I'm confused by steps 2-5. Am I starting the server through the terminal or Appium Desktop? And what do you mean by connect to the running appium server?

**Comment by jlee6890** [comment]
Nevermind, got it to work. Thanks!

**Comment by parikhparth23** [comment]
please follow the steps mentioned by @KazuCocoa above.
In "point 4" when you open Appium on the status bar on mac click on "Appium"-> "New Session Window"-> "Enter you appium server ip address and the port number"-> "add desired capabilities"

**Comment by KazuCocoa** [comment]
👍
The latest desktop app has been released which bundles Appium 1.12.1
